R/convert_year_types.R

```diff
@@ -0,0 +1,46 @@
+#' Convert year types - Financial year form to the alternate form
+#' 
+#' Converts year type from the financial year form '1718' to the year form '2017'.
+#'
+#' @param fyyear vector of financial years in the form '1718'
+#'
+#' @return a vector of years in the alternate form '2017'
+#' @export
+#'
+#' @examples
+#' fyyears <- c("1718", "1819")
+#' convert_fyyear_to_year(fyyears)
+#' [1] "2017" "2018"
+convert_fyyear_to_year <- function(fyyear) {
+  for (i in 1:length(fyyear)) {
+    if (substr(fyyear[i], 1, 2) > substr(fyyear[i], 3, 4)) {
+      stop ("Year has been entered in the wrong format, try again using form `1718` or use function `convert_year_to_fyyear` to convert to the financial year form.")
+  }}
+  year <- paste0("20", substr(fyyear, 1, 2))
+  return(year)
+}
+
+
+
+
+#' Convert year types - Alternate year form to financial year form
+#' 
+#' Convert a year type from alternate form '2017' to normal financial year form '1718'.
+#'
+#' @param year vector of years in the form '2017'
+#'
+#' @return a vector of years in the normal financial year form '1718'
+#' @export
+#'
+#' @examples
+#' years <- c("2017", "2018")
+#' convert_year_to_fyyear(years)
+#' [1] "1718" "1819"
```

Comment:
Remove this line as above, please.

Reply:
Just the [1] line or calling the function as well?

Reply:
Yep just the 'output' `[1]` line as you've done 👍